GlyssenEngineTests/Properties/Resources.Designer.cs

```diff
@@ -378,6 +378,16 @@ internal class Resources {
             }
         }
         
+        /// <summary>

```

Comment:
Technically, there is no reason to add this as a resource, since we are not compiling the tests that would need it. It probably doesn't hurt anything, though.

Reply:
I left it. It will be available if and when needed.